lib/modules/commentTools.js

```diff
@@ -298,114 +298,132 @@ modules['commentTools'] = {
 				this.fetchTitle();
 			}
 
+			fastdom.write(function() {
+				$body.on('click', 'div.markdownEditor-wrapper a:not(.userTagLink)', function(e) {
```

Comment:
I don't know exactly why I put one there, haha. I think it might have been the fact it's got an element selector there, but it should be in the read cycle instead